### ISSUE TITLE
San Francisco IPFS

### ISSUE THREAD

**Issue by jbenet** [ipfs/community#53]
@diasdavid and I will be in SF/PA from 10/19-10/24. We will host a meetup, likely on 2015-10-20 as I cannot do any other day in the range. Location + Time still TBD. Will use this issue to coordinate.

**Comment by typesend** [comment]
Great!

**Comment by amstocker** [comment]
Flying into SFO at 8pm on 10/20!  I could uber from the airport if this is going to be a late night kind of thing :)

**Comment by jbenet** [comment]
cool!

we would likely run till late, though meetups should start 7/8pm weeknights.
>
>

**Comment by jbenet** [comment]
@diasdavid would you mind setting up a meetup page for this? we should find a venue, i'll ask friends.

**Comment by jbenet** [comment]
meetup or attending, whatever

**Comment by diasdavid** [comment]
On it (contacting meetup.com support to let me open more than 3 meetups in one account)

**Comment by RichardLitt** [comment]
Reminder; these need to be added to the community calendar as soon as dates are set in stone. We can update the events with locations and exact times as they appear.

**Comment by diasdavid** [comment]
@RichardLitt added the Portland one :) will add the Bay area one once I get manage to get the group created

**Comment by jbenet** [comment]
> meetup.com, i'm getting tired of your shit.
> -- everyone ever

**Comment by diasdavid** [comment]
Created http://attending.io/events/ipfs-san-francisco-october-2015

**Comment by harlantwood** [comment]
Awesome!  @aquabu do we have the capacity to host this at [WeWork in downtown SF](https://goo.gl/maps/2ZYp9A1RX542)?  Would that location work for you @jbenet @diasdavid?

**Comment by harlantwood** [comment]
I will contact WeWork to try to reserve if you like the sound of this venue... here is their marketing site: https://www.wework.com/locations/san-francisco/transbay

**Comment by diasdavid** [comment]
Thank you @harlantwood WeWork looks awesome! I've just arrived at SF (@jbenet arrived this morning too) let us know if you need any extra info from us for the booking.

**Comment by harlantwood** [comment]
I left a message yesterday but didn't hear back, will try again today.

**Comment by harlantwood** [comment]
The Transbay WeWork does not have space available for Tues evening.  I emailed the other bay area locations to inquire about another location; have not heard back yet.

If other people have other location ideas, please pursue them, as WeWork does not seem toooo promising at the moment.

**Comment by diasdavid** [comment]
Thank you anyways @harlantwood  :)

@jbenet you had a couple of places in mind, want to check those? Otherwise I can start querying some friends

**Comment by harlantwood** [comment]
Sure NP.

**Comment by tilgovi** [comment]
Any guess about the capacity needed?

**Comment by jbenet** [comment]
Our friends at Sourcegraph offered to host us. Did you have a place in mind @tilgovi ? what place?

**Comment by tilgovi** [comment]
I did not.

**Comment by diasdavid** [comment]
We got a venue! Thanks Sourcegraph

All details on: http://attending.io/events/ipfs-san-francisco-october-2015

**Comment by diasdavid** [comment]
@tmatcha wants to help us run the local SF meetup :)

**Comment by jbenet** [comment]
hey @tmatcha thanks for the offer to help. did we meet at the meetup last tuesday?

**Comment by RichardLitt** [comment]
Closing. Reopen or create a new issue if another meetup is planned.

**Comment by alan-andrade** [comment]
Any plans for another meetup around SF ?

**Comment by whyrusleeping** [comment]
Would be pretty cool to do

**Comment by harlantwood** [comment]
Yeah!  👍

**Comment by RichardLitt** [comment]
I might be there in mid-June.

**Comment by alan-andrade** [comment]
Awesome ! Looking forward for it.

**Comment by dryajov** [comment]
I migth be able to make it.

**Comment by Fil** [comment]
Any chance of doing something in SF between 8 September and 21 Sep, 2017? (I'll be visiting from France.)

**Comment by diasdavid** [comment]
If anyone interested in leading an IPFS SF meetup, please check https://github.com/ipfs/community/issues/283

**Comment by smitthakkar1** [comment]
is there any slack/discord group of people who are working with IPFS ?

**Comment by VictorBjelkholm** [comment]
> is there any slack/discord group of people who are working with IPFS ?

No, we use IRC for communications in the community as it's a open protocol everyone can use and implement. You can find us on Freenode in #ipfs (general) and #ipfs-dev (for development). If you don't have a IRC client, you can use this link: https://webchat.freenode.net/?channels=%23ipfs

**Comment by tfauzia** [comment]
Hello..i want asking something. i have final project in my college this is about memory optimization analysis in ipfs virtualization. 
So my teacher ask to me for searching tools that can use for measuring memory when using ipfs and then make a conclusion about optimalization memory for ipfs. Do you know that tools or can you are give me some suggestion about that tools? Thank you so much

**Comment by whyrusleeping** [comment]
Hey @tfauzia You can get the memory usage as reported by go if you use `go tool pprof localhost:5001/debug/pprof/heap` while the daemon is running. That number will be slightly different than reported by the OS (say, by looking at `ps aux | grep ipfs`) because the go runtime doesn’t give memory back to the OS as often as it frees it up itself.

**Comment by Stebalien** [comment]
Y'all, meetup on may 9th: https://www.meetup.com/San-Francisco-IPFS/events/261077320/

**Comment by Stebalien** [comment]
There are quite a few interested users subscribed to these threads. I'll do a bit of cleanup.

**Comment by Stebalien** [comment]
Hm. Actually, this issue _was_ created for a specific meetup. Let's just organize on the meetup page.

**Comment by Stebalien** [comment]
(nevermind, let's keep this open but modify the issue)

**Comment by aquabu** [comment]
Having updated info in the community repo would be helpful - for instance the link to the calendar of events, contribution guidelines and code of conduct are helpful. And there are lots of repo watchers & stars which indicates interest in getting this info on GitHub.

I'm on this thread from 2016 and even though the repo is out of date it actually helped answer some questions for me that I had at Dweb camp about how to stay involved with the IPFS community :)

**Comment by leshokunin** [comment]
@Stebalien let's have a quick chat at the Meetup about how to make this better!
